src/stapi_fastapi/types/json_schema_model.py

```diff
@@ -8,7 +8,7 @@
 )
 
 
-def validate(v: Any):
+def validate(v: Any) -> Any:
```

Comment:
I just ran into a problem with `Any` in a different context and it made me learn some things about what `Any` means and now I'm wary when I see it. Of particular help on this subject were [these mypy docs on dynamic typing behaviors](https://mypy.readthedocs.io/en/stable/dynamic_typing.html).

In short, I _think_ better typing for this function would be like this:

```suggestion
def validate(v: object) -> type[BaseModel]:
```

That said, with how this function is used I don't know that the typing being better is a concern. This is more an FYI comment for something I recently learned and found interesting than a request for a change.